### ISSUE TITLE
9.4.1.2/4.3 All buttons in the image editor (see screenshot) used for editing an image have not been implemented as such. It is best to use native HTML elements such as "button" or the appropriate ARIA role role="button" for implementation. If ARIA is used, the element should be made both focusable with tabindex="0" and keyboard-activatable. Additionally, SVG elements should be hidden from screen readers using the "aria-hidden" attribute, and the "aria-selected" attribute should not be used in this case.

### ISSUE THREAD

**Issue by szaimen** [nextcloud/server#42233]
https://report.bitvtest.de/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v4-n3

9.4.1.2/4.3 All buttons in the image editor (see screenshot) used for editing an image have not been implemented as such. It is best to use native HTML elements such as "button" or the appropriate ARIA role role="button" for implementation. If ARIA is used, the element should be made both focusable with tabindex="0" and keyboard-activatable. Additionally, SVG elements should be hidden from screen readers using the "aria-hidden" attribute, and the "aria-selected" attribute should not be used in this case.

![image](https://github.com/nextcloud/server/assets/42591237/5a6fff84-a5a2-414a-a553-8770bd405da5)
![image](https://github.com/nextcloud/server/assets/42591237/073012fd-6c79-4a43-b1b4-3a79ff9ffd11)

**Comment by JuliaKirschenheuter** [comment]
duplicate of https://github.com/nextcloud/server/issues/41898
